setup.py

```diff
@@ -180,6 +180,7 @@ def write_version(filename=os.path.join(*['airflow',
     'google-cloud-bigtable==0.31.0',
     'google-cloud-spanner>=1.7.1',
     'google-cloud-vision>=0.35.2',
+    'google-cloud-translate>=1.3.3',
```

Comment:
`google-cloud-translate>=1.3.3` before `google-cloud-vision>=0.35.2` for alphabetical?

Reply:
Good catch

Reply:
Fixed.